api/controllers/v0.3/endpoints.js

```diff
@@ -283,7 +283,8 @@ async function createPatron(req, res) {
     });
 
   let address = new Address(req.body.address);
-  const policy = Policy({ policyType: "webApplicant" });
+  // TODO: What should the default policy be?
+  const policy = Policy({ policyType: req.body.policyType || "simplye" });
```

Comment:
What happened to "webApplicant"?

Reply:
It's still available but the requirement might change on what's the default.